specification/sdk-tracing.md

```diff
@@ -260,6 +260,53 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter. These spans are called _unfinished_ spans.
+On each interval defined by `reportIntervalMillis`, all unfinished spans that
+have been started more than `reportIntervalMillis` ago, are exported.
+
+**Configurable parameters:**
+
+* `exporter` - the exporter where the spans are pushed.
+* `maxWatchedSpans` - the maximum number of unfinished spans to be watched.
+  While reached, no new spans will be accepted. The default value is `2048`.
+* `reportIntervalMillis` - the delay interval in milliseconds between two
+  consecutive exports. The default value is `5000`.
+* `exporterTimeoutMillis` - how long the export can run before it is cancelled.
+  The default value is `30000`.
+* `maxExportBatchSize` - the maximum batch size of every export. It must be
+  smaller or equal to `maxWatchedSpans`. The default value is `512`.
+* `exportEndedSpans` (boolean) - if set, spans are also exported once they are
+  ended.
+  The default value is `false`.
```

Comment:
I would use `true` as default value here as I assume most users are interested on finished spans also.

Reply:
This depends on the use case. If I use it for debugging purposes like the zPages feature of OpenCensus I will likely have a default exporter for finished spans already in place and will use the span watcher only for running spans.

Reply:
Don't know what zPages are. But even for debugging I would like to have open and finished spans in one (e.g. log) exporter.